include/neu_errcodes.h

```diff
@@ -94,6 +94,13 @@ typedef enum {
     NEU_ERR_LIBRARY_INFO_INVALID  = 2302,
     NEU_ERR_LIBRARY_NAME_CONFLICT = 2303,
 
+    NEU_ERR_LICENSE_NOT_FOUND = 2400,
+    NEU_ERR_LICENSE_INVALID   = 2401,
+    NEU_ERR_LICENSE_EXPIRED   = 2402,
+    NEU_ERR_LICENSE_DISABLED  = 2403,
```

Comment:
what different with invalid

Reply:
invalid for indicating the license is invalid in format,
disabled is for indicating a certain plugin is disabled/enabled.